tests/network/example_fuji_network.go

```diff
@@ -0,0 +1,146 @@
+package network
+
+import (
+	"context"
+	"crypto/ecdsa"
+	"math/big"
+	"time"
+
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/subnet-evm/core/types"
+	"github.com/ava-labs/subnet-evm/ethclient"
+	"github.com/ava-labs/teleporter/tests/utils"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+	. "github.com/onsi/gomega"
+)
+
+// Amplify, Bulletin, Conduit subnet constants
+var (
+	teleporterContractAddress = common.HexToAddress("0x50A46AA7b2eCBe2B1AbB7df865B9A87f5eed8635")
+
+	amplifySubnetID     ids.ID
+	amplifyBlockchainID ids.ID
+
+	bulletinSubnetID     ids.ID
+	bulletinBlockchainID ids.ID
+
+	conduitSubnetID     ids.ID
+	conduitBlockchainID ids.ID
+
+	userAddress = common.HexToAddress("0xF0Aa98fDE5f1d08F0CCEC68A7d7A7Eae31c5E9C9")
+	skHex       = "" // To be supplied by user
+)
+
+func init() {
+	var err error
+
+	amplifySubnetID, err = ids.FromString("2PsShLjrFFwR51DMcAh8pyuwzLn1Ym3zRhuXLTmLCR1STk2mL6")
+	if err != nil {
+		panic(err)
+	}
+	amplifyBlockchainID, err = ids.FromString("2nFUad4Nw4pCgEF6MwYgGuKrzKbHJzM8wF29jeVUL41RWHgNRa")
+	if err != nil {
+		panic(err)
+	}
+
+	bulletinSubnetID, err = ids.FromString("cbXsFGWSDWUYTmRXUoCirVDdQkZmUWrkQQYoVc2wUoDm8eFup")
+	if err != nil {
+		panic(err)
+	}
+	bulletinBlockchainID, err = ids.FromString("2e3RJ3ub9Pceh8fJ3HX3gZ6nSXJLvBJ9WoXLcU4nwdpZ8X2RLq")
+	if err != nil {
+		panic(err)
+	}
+
+	conduitSubnetID, err = ids.FromString("wW7JVmjXp8SKrpacGzM81RBXdfcLDVY6M2DkFyArEXgtkyozK")
+	if err != nil {
+		panic(err)
+	}
+	conduitBlockchainID, err = ids.FromString("9asUA3QckLh7vGnFQiiUJGPTx8KE4nFtP8c1wTWJuP8XiWW75")
+	if err != nil {
+		panic(err)
+	}
+}
+
+// Implements Network, pointing to subnets deployed on Fuji
+type FujiNetwork struct{}
+
+func (g *FujiNetwork) GetSubnetsInfo() []utils.SubnetTestInfo {
```

Comment:
Nit: why `g` here? Could we make it `fj` for `FujiNetwork`?

Reply:
Changed both this and the `LocalNetwork` receiver variable to `n` for network.

(The `g` came from `GinkgoNetwork`, which I changed to `LocalNetwork`)